packages/broker/src/plugins/operator/OperatorPlugin.ts

```diff
@@ -27,8 +26,7 @@ export interface OperatorPluginConfig {
 }
 
 export interface OperatorServiceConfig {
-    provider: Provider
-    signer: Signer
+    nodeWallet: Wallet
```

Comment:
Can't the operator have many node wallets? Or is the idea of this naming to emphasize it's going to be the "node's wallet" and not the operator's?

Reply:
This is the wallet of the Broker node, not operator's (or operator contract's) wallet.